src/Game.java

```diff
@@ -3,46 +3,47 @@
 public class Game {
 
     private Rules rules;
-    private SYMBOL aiPlayer;
-    private SYMBOL winningSymbol;
+    private Symbol aiPlayer;
+    private Symbol winningSymbol;
 
-    public Game() {
+    public Game(Rules rules) {
         aiPlayer = null;
-        rules = new Rules();
-        SYMBOL.setOrdinalToSymbol();
+        this.rules = rules;
+        Symbol.setOrdinalToSymbol();
     }
 
-    public SYMBOL generateAIMove() {
-        int range = SYMBOL.ordinalToSymbol.size();
+    public Symbol generateAIMove() {
+        int range = Symbol.ordinalToSymbol.size();
         aiPlayer = getRandomSymbolInRange(range);
         return aiPlayer;
```

Comment:
Is this field used anywhere else? Again, avoid fields if possible, especially if they can be reassigned.